tm2/pkg/bft/types/params.go

```diff
@@ -16,6 +16,18 @@ const (
 
 	// MaxBlockPartsCount is the maximum count of block parts.
 	MaxBlockPartsCount = (MaxBlockSizeBytes / BlockPartSizeBytes) + 1
+
+	// MaxBlockTxBytes is the max size of the block transaction
+	MaxBlockTxBytes int64 = 1000000 // 1MB
+
+	// MaxBlockDataBytes is the max size of the block data
+	MaxBlockDataBytes int64 = 2000000 // 2MB
+
+	// MaxBlockMaxGas is the max gas limit for the block
+	MaxBlockMaxGas int64 = 10000000 // 10M gas
+
+	// BlockTimeIotaMS is the block time iota (in ms)
+	BlockTimeIotaMS int64 = 100 // ms
```

Comment:
why changing from 1000 to 100?

Reply:
actually, i think it makes sense; but it would be nice that you let a clear comment in the PR to explain the 4 value changes.

Reply:
It is the value I took from the `gnoland start` command, the previous defaults were never actually being used